packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -33,9 +33,41 @@
   overflow: hidden;
 }
 
+.mdc-chip--with-leading-icon {
```

Comment:
Is it not possible to accomplish this via padding or margin on the icons themselves, rather than needing to add one or two classes to every chip in addition to adding the icon?

Reply:
Ohh so negative margins are a thing. Will do!